pimd/pim6_cmd.h

```diff
@@ -57,6 +57,7 @@
 #define DEBUG_PIMV6_PACKETDUMP_RECV_STR "Dump received packets\n"
 #define DEBUG_PIMV6_TRACE_STR "PIMv6 internal daemon activity\n"
 #define DEBUG_PIMV6_ZEBRA_STR "ZEBRA protocol activity\n"
+#define DEBUG_MROUTE6_STR "PIM6 interaction with kernel MFC cache\n"
```

Comment:
PIMv6 (I assume we should use the same everywhere, a quick shot two lines above :smile: )